### ISSUE TITLE
missing processing jars

### ISSUE THREAD

**Issue by boggle** [n8han/spde-examples#2]
Hi, 

i wanted to try out spde but didnt get very far. git cloning spde-sbt and spde-examples both give me 

```
    [error] /Users/gonzo/Current/spde-examples/Flocking/src_managed/main/scala/glob.scala:1: not found: value processing
    [error] import processing.core._
    [error]        ^
    [error] /Users/stepn/Current/spde-examples/Flocking/src_managed/main/scala/glob.scala:9: too many arguments for constructor Object: ()java.lang.Object
    [error]   lazy val px = new DrawProxy(this) {
    [error]                     ^
    [error] two errors found
```

somehow the processing jars are not downloaded  to the right places.

I am using java 1.6 on mac os 10.6

**Comment by n8han** [comment]
Hi boggle,

You probably need to run "update" in sbt. This will fetch the dependencies.

**Comment by boggle** [comment]
which i did yesterday.  today it works, oh well... maybe my net connection was down and sbt didnt report that properly.  still, there are missing artifacts:

```
    sbt update
```

...

```
    [warn]  ::::::::::::::::::::::::::::::::::::::::::::::
    [warn]  ::          UNRESOLVED DEPENDENCIES         ::
    [warn]  ::::::::::::::::::::::::::::::::::::::::::::::
    [warn]  :: gstreamer.java#gstreamer-java;1.2: not found
    [warn]  :: gsvideo#gsvideo;0.6-pre0: not found
    [warn]  ::::::::::::::::::::::::::::::::::::::::::::::
    [info] 
    [warn] :: problems sum
```

...

```
    [info] == Geometry / update ==
    [error] Error running <anonymous>: sbt.ResolveException: unresolved dependency: gstreamer.java#gstreamer-java;1.2: not found
    [error] unresolved dependency: gsvideo#gsvideo;0.6-pre0: not found
    [error] Error running <anonymous>: sbt.ResolveException: unresolved dependency: gstreamer.java#gstreamer-java;1.2: not found
    [error] unresolved dependency: gsvideo#gsvideo;0.6-pre0: not found
    [info] 
    [info] Total time: 101 s, completed 21.08.2010 16:18:05
```

**Comment by boggle** [comment]
works now, i changed the dependencies in `project/plugins/src_managed/spde-sbt-plugin-0.4.0/VideoProject.scala' to

```
     val jna = "com.sun.jna" % "jna" % "3.0.9"
     val gstreamerJava = "com.googlecode.gstreamer-java" % "gstreamer-java" % "1.4"
```

This almost did it.  Since I'm using homebrew, I needed an additional 

```
    ln -s /usr/local/lib /Users/gonzo/Current/spde-examples/Scratch/lib_managed/scala_2.8.0/compile//gstreamer/macosx
```

**Comment by n8han** [comment]
So the gstreamer artifacts are available though a central repository now? If that's the case I'll certainly change the dependencies.

And are you able to get the video demo to work on Mac? That would be impressive. I think I got it to work on a Mac once. It's difficult enough that I figure I'll have to reimplement quicktime support through rococoa at some point.

**Comment by n8han** [comment]
I updated all the gs-\* dependencies and published new versions, 0.4.2 for spde-sbt-plugin. Seems to be updating much better now that more of the dependencies are standard maven ones and not hardcoded "from" references.
